.travis.yml

```diff
@@ -22,4 +22,5 @@ branches:
   - /^v\d+\.\d+\.\d+$/
   only:
   - master
```

Comment:
Should we remove master?

Reply:
let's wait to remove it until the master branch is actually removed